src/material-experimental/mdc-table/table.scss

```diff
@@ -27,3 +27,18 @@
 mat-row.mat-mdc-row, mat-header-row.mat-mdc-header-row, mat-footer-row.mat-mdc-footer-row {
   border-bottom: none;
 }
+
+// Cells need to inherit their background in order to overlap each other when sticky.
+// The background needs to be inherited from the table, tbody/tfoot, row
+// (already set in MDC), and cell.
+.mat-mdc-table tbody, .mat-mdc-table tfoot,
+.mat-mdc-cell, .mat-mdc-footer-cell {
+  background: inherit;
+}
+
+// Disable hover styling while MDC uses an opacity for its color.
+// When the hover style is used with sticky cells, the opacity shows the cells overlapping.
+.mat-mdc-table .mat-mdc-row:hover,
+.mat-mdc-table .mat-mdc-footer-row:hover {
```

Comment:
Could you combine these selectors with the ones above?

Reply:
I'd like them separate because it could be safely removed if (1) MDC fixes their hover color or (2) we make some extra input or feature to re-enable the hover styles. However, the above styles shouldn't be removed (well unless MDC adds the inherit background themselves).